openrtb/auctiontype.go

```diff
@@ -3,11 +3,13 @@ package openrtb
 // AuctionType type denotes the mode of the auction in which the ad exchange server will run.
 // 1 = First Price.
 // 2 = Second Price.
-// See OpenRTB 2.3.1 Sec 3.2.1.
+// 3 = Use bid floor as the deal price.
+// See OpenRTB 2.5 Sec 3.2.1.
```

Comment:
```suggestion
// See OpenRTB 2.5 Sec 3.2.1 and 3.2.12
```

Reply:
Have make DealAuctionType and BidAuctionType to avoid confusing.